src/pages/attendance-history.tsx

```diff
@@ -790,7 +790,7 @@ const UserAttendanceHistory = () => {
             onClose={handleClose}
```

Comment:
@suvarnakale please add screenshot and update commit message with respective task id

Reply:
This has no task id, actually it was a bug but it was not logged.